Classes/ViewHelpers/Data/SplitViewHelper.php

```diff
@@ -38,18 +38,22 @@
 class SplitViewHelper extends AbstractViewHelper
 {
     /**
-     * @param string $string    The string to split into components
-     * @param string $separator The string separating the components
-     *
-     * @return string|int|bool|array
+     * Register arguments.
+     */
+    public function initializeArguments()
+    {
+        parent::initializeArguments();
+        $this->registerArgument('string', 'string', 'The string to split into components', false, null);
+        $this->registerArgument('separator', 'string', 'The string separating the components', true);
```

Comment:
There is no default separator, before we hat `, ` as separator. A default value can be added as  parameter to the argument registration.

Reply:
done